requirements.txt

```diff
@@ -1,6 +1,60 @@
-botocore
-boto3>=1.4.3
+arrow==0.12.1
+asn1crypto==0.24.0
+aws-sam-cli==0.4.0
+aws-sam-translator==1.6.0
+backports.functools-lru-cache==1.5
+backports.ssl-match-hostname==3.5.0.1
+beautifulsoup4==4.6.0
+binaryornot==0.4.4
+boto3==1.7.20
+botocore==1.10.20
+certifi==2018.4.16
+cffi==1.11.5
+chardet==3.0.4
```

Comment:
shouldn't cfn-lint be in this list too ?

Reply:
Not sure why pip freeze does not pick it up I will add it manually

Reply:
needed to use pip3, doh! (corrected)